src/Schema/SQLServerSchemaManager.php

```diff
@@ -375,6 +383,21 @@ private function getDatabaseCollation(): string
         return $this->databaseCollation;
     }
 
+    protected function selectTableNames(string $databaseName): Result
+    {
+        // The "sysdiagrams" table must be ignored as it's internal SQL Server table for Database Diagrams
+        $sql = <<<'SQL'
+SELECT name,
+       SCHEMA_NAME(schema_id) AS schema_name
+FROM sys.objects
```

Comment:
We're moving from `sysobjects` in `SQLServerPlatform` to `sys.objects` here since the former is deprecated.